app/controllers/works_controller.rb

```diff
@@ -210,19 +210,18 @@ def drafts
   def show
     @tag_groups = @work.tag_groups
     if @work.unrevealed?
-      @page_title = ts('Mystery Work')
+      @page_title = ts("Mystery Work")
     else
-      page_title_inner = ''
-      page_creator = ''
-      if @work.anonymous?
-        page_creator = ts('Anonymous')
-      else
-        page_creator = @work.pseuds.collect(&:byline).sort.join(', ')
-      end
-      page_title_inner = if @tag_groups['Fandom'].size > 3
-                           ts('Multifandom')
+      page_creator = if @work.anonymous?
+                       ts("Anonymous")
+                     else
+                       @work.pseuds.map(&:byline).sort.join(", ")
+                     end
+      fandoms = @tag_groups["Fandom"]
+      page_title_inner = if fandoms.size > 3
```

Comment:
This precisely mimics the existing behaviour, though now I look at it, this means that: 
- if there are 4 fandoms, you will get "Multifandom"
- if you have 1 fandom, you will get that fandom
- if you have 2 or 3 fandoms, you will only get the first fandom
Maybe I should take this opportunity to change this line to `if fandoms.size > 1`